ETC.tsv

```diff
@@ -5380,7 +5380,7 @@ ETC_20150317_005379	Good Job!
 ETC_20150317_005380	Where are you!!!
 ETC_20150317_005381	Come out!
 ETC_20150317_005382	Thorny Gateway
-ETC_20150317_005383	Bag that contains something
+ETC_20150317_005383	$Bag with Something Inside
```

Comment:
The Korean text literally says Bag with something inside, just trying to keep as much meaning as possible.

We can decide if "Bag" works during localization.